superset/__init__.py

```diff
@@ -222,9 +222,11 @@ def is_feature_enabled(feature):
 if conf.get("ENABLE_FLASK_COMPRESS"):
     Compress(app)
 
+
+talisman = Talisman()
+
 if app.config["TALISMAN_ENABLED"]:
-    talisman_config = app.config.get("TALISMAN_CONFIG")
-    Talisman(app, **talisman_config)
+    talisman.init_app(app, **app.config["TALISMAN_CONFIG"])
```

Comment:
We don't need to use `get(...)` here since the config key exists in config.py.

Reply:
Makes sense. Also, this code would have broken before if there was no config anyway because we were doing `**` on a None

Reply:
@etr2460 correct.